app/models/cohort_columns/days_since_cas_match.rb

```diff
@@ -0,0 +1,26 @@
+###
+# Copyright 2016 - 2019 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module CohortColumns
+  class DaysSinceCasMatch < ReadOnly
+    attribute :column, String, lazy: true, default: :days_since_cas_match
+    attribute :translation_key, String, lazy: true, default: 'Days Since CAS Match'
+    attribute :title, String, lazy: true, default: -> (model, attr) { _(model.translation_key)}
+
+    def value(cohort_client)
+      match_started_time = cohort_client.client.cas_reports.maximum(:match_started_at)
```

Comment:
This will add N queries to loading a cohort.  Since we sometimes have cohorts of 1000+ clients, we need to pre-cache this.  See `app/models/cohort_columns/vispdat_priority_score.rb` for an example of how we'll need to pre-calculate.  These are calculated in `app/models/grda_warehouse/warehouse_clients_processed.rb`